### PR TITLE
extra logging around github cache errors

### PR DESCRIPTION
**Pull request by samlavery**
*Additional logs to STDERR in the event we see an issue during clone/build from github*
Small change, should only add a few extra log lines.

### CODE REVIEW

File: services/ext/github/repos.go [hunk 1]
```diff
@@ -52,19 +52,28 @@ func (s *Repos) Get(ctx context.Context, repo string) (*sourcegraph.RemoteRepo,
 	owner, repoName, err := githubutil.SplitRepoURI(repo)
 	if err != nil {
 		reposGithubPublicCacheCounter.WithLabelValues("local-error").Inc()
+		log15.Error("github cache-get local-error for repo ", repoName, "err", err)
```
Comment (rothfels) [inline services/ext/github/repos.go]: The args after the msg should be key-value pairs. So insert a string arg "repo" as the 2nd arg. See The args after the msg should be key-value pairs. So insert a string arg "repo" as the 2nd arg. (you also don't need to end the message string with a space)

File: services/ext/github/repos.go [hunk 2]
```diff
@@ -52,19 +52,28 @@ func (s *Repos) Get(ctx context.Context, repo string) (*sourcegraph.RemoteRepo,
...
 		return nil, grpc.Errorf(codes.NotFound, "github repo not found: %s", repo)
 	}
 
 	ghrepo, resp, err := client(ctx).repos.Get(owner, repoName)
 	if err != nil {
 		reposGithubPublicCacheCounter.WithLabelValues("error").Inc()
+
+		log15.Error("github cache-get standard error for repo ", repoName, "err", err)
```
Comment (rothfels) [inline services/ext/github/repos.go]: Et al

**Comment by keegancsmith** [comment]
We have this under control now. Should we just abandon this?

**Comment by rothfels** [comment]
Closing this, @samlavery feel free to reopen if still relevant.

**Pull request closed without merging**